src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -1571,6 +1571,87 @@ def cli(self, ip,output=None):
         return interface_dict
 
 
+class ShowIpv6InterfaceBriefPipeIpSchema(MetaParser):
+    """Schema for show ipv6 interface brief | include <WORD>"""
+    schema = {'interface':
+                {Any():
+                    {Optional('ipv6_address'): str,
+                    Optional('interface_status'): str,
+                    Optional('protocol_status'): str}
+                },
+            }
+
+
+class ShowIpv6InterfaceBriefPipeIp(ShowIpv6InterfaceBriefPipeIpSchema):
+    """Parser for:
+     show ipv6 interface brief | include <WORD>
+     parser class implements detail parsing mechanisms for cli
+
+     In reality, the 'include' would only show the IPv6 address without
+     interface information. This class mimics the IPv4 behavior by executing
+
+     show ipv6 interface brief
+
+     And parsing the address <> interface name mapping to generate a similar
+     output. Also note that because link-local addresses may be statically
+     defined for interfaces, and every interface can have the same LL
+     address, this implementation supports parsing for fe80:: addresses
+     and will return every matching interface.
+    """
+    #*************************
+    # schema - class variable
+    #
+    # Purpose is to make sure the parser always return the output
+    # (nested dict) that has the same data structure across all supported
+    # parsing mechanisms (cli(), yang(), xml()).
+
+    cli_command = 'show ipv6 interface brief'
+
+    def cli(self, ip=None, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+        interface_dict = {}
+
+        # GigabitEthernet1       [up/up]
+        #     FE80::5054:FF:FE18:9003
+        #     2001:DB8:C15:C0::10
```

Comment:
Please align the comments with the regex compile statements, e.g. line 1 for p1, line 2-3 for p2

Reply:
Updated comment locations to represent the corresponding regular expression